### PR TITLE
Update Add PHI

### PR DESCRIPTION
**Pull request by Phinetwork**
Add PHI Network Details To Snapshot

Fixes # .

Changes proposed in this pull request:
Added PHI 
- 
-

### CODE REVIEW

File: src/networks.json [hunk 1]
```diff
@@ -108,6 +108,21 @@
...
     "logo": "ipfs://Qmec3HLoN4QhwZAhw4XTi2aN8Wwmcko5hHN22sHARzb9tw"
   },
+  "4181": {
+    "key": "4181",
+    "name": "PHI",
+    "chainId": 4181,
+    "network": "PHI",
+    "multicall": "0xB141139f17c9F6B6B0da82bfC8E1153570AD44fE",
+    "rpc": [
+      "https://rpc1.phi.network"
```
Comment (ChaituVR) [inline src/networks.json:118]: Hey @Phinetwork Looks like this is not a full archive node?? try passing older block numbers

Comment (ChaituVR) [reply src/networks.json:118]: It give this error:
```
missing trie node b2409a4e98f07e15137e31d89255a9707cbd45ab07baefb2c540c46702851208
```

File: src/networks.json [hunk 2]
```diff
@@ -108,6 +108,21 @@
     "explorer": "https://ubiqscan.io",
     "logo": "ipfs://Qmec3HLoN4QhwZAhw4XTi2aN8Wwmcko5hHN22sHARzb9tw"
   },
+  "4181": {
+    "key": "4181",
+    "name": "PHI",
+    "chainId": 4181,
+    "network": "PHI",
+    "multicall": "0xB141139f17c9F6B6B0da82bfC8E1153570AD44fE",
```
Comment (ChaituVR) [inline src/networks.json:116]: Is there a way to check the code here?
https://explorer.phi.network/address/0xB141139f17c9F6B6B0da82bfC8E1153570AD44fE

**Review by ChaituVR — CHANGES_REQUESTED** [review]

File: src/networks.json (hunk 1 above)
Comment (Phinetwork) [reply src/networks.json:118]: Node Provided is a full archive node.

File: src/networks.json (hunk 2 above)
Comment (Phinetwork) [reply src/networks.json:116]: Can See Full Documentation[ Here ](https://docs.phi.network)

**Comment by Phinetwork** [comment]
All Checks Have Passed Can We Please Add This Thank You

**Comment by Phinetwork** [comment]
all checks passed can we please add this also has been added to chainlist https://github.com/snapshot-labs/snapshot.js/runs/6645109247?check_suite_focus=true 

https://github.com/ethereum-lists/chains/pull/1200

File: src/networks.json (hunk 1 above)
Comment (ChaituVR) [reply src/networks.json:118]: Can you try calling it with older block numbers like 10, 100, 1000 etc

File: src/networks.json (hunk 2 above)
Comment (ChaituVR) [reply src/networks.json:116]: Can find source code of this contract here use the first PR to make any changes https://github.com/snapshot-labs/snapshot.js/pull/591

**Review by ChaituVR — CHANGES_REQUESTED** [review]
Hi @Phinetwork Look at the reviews above, i still get this error:
```
missing trie node b2409a4e98f07e15137e31d89255a9707cbd45ab07baefb2c540c46702851208
```

**Comment by Phinetwork** [comment]
Screen Shot 2022-06-19 at 9 38 47 PM

It says all checks have passed not sure what your talking about what do you need from us. this is a full archive node.

**Comment by ChaituVR** [comment]
@Phinetwork Try testing with older blocks, example `392631`

**Comment by Phinetwork** [comment]
can we merge this pull request now

**Comment by ChaituVR** [comment]
@Phinetwork This is not a full archive node, even if we merge this, the node will fail to work as per snapshot requirements,

This is the example request you can try:
```bash
curl 'https://rpc1.phi.network/' \
  -H 'Accept: */*' \
  -H 'Accept-Language: en-US,en;q=0.9' \
  -H 'Cache-Control: max-age=0' \
  -H 'Connection: keep-alive' \
  -H 'Sec-Fetch-Dest: empty' \
  -H 'Sec-Fetch-Mode: cors' \
  -H 'Sec-Fetch-Site: cross-site' \ Safari/537.36' \
  -H 'content-type: application/json' \
  --data-raw '{"method":"eth_getBalance","params":["0xb141139f17c9f6b6b0da82bfc8e1153570ad44fe","0x1"],"id":44,"jsonrpc":"2.0"}' \
  --compressed
```

This is the error:
```bash
Error: processing response error (body="{\"jsonrpc\":\"2.0\",\"id\":44,\"error\":{\"code\":-32000,\"message\":\"missing trie node b2409a4e98f07e15137e31d89255a9707cbd45ab07baefb2c540c46702851208 (path )\"}}\n", error={"code":-32000}, requestBody="{\"method\":\"eth_getBalance\",\"params\":[\"0xb141139f17c9f6b6b0da82bfc8e1153570ad44fe\",\"0x1\"],\"id\":44,\"jsonrpc\":\"2.0\"}", requestMethod="POST", url="https://rpc1.phi.network", code=SERVER_ERROR, version=web/5.6.0)
```

**Pull request closed without merging**